provision.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+set -x
+
+cd ~vagrant/src
+
+# apt-get update
+
+export LC_ALL="en_US.UTF-8"
+locale-gen en_US.UTF-8
+
+# Install install Apache bits
+apt-get install -y -q libapache2-mod-wsgi
+
+# Install dev things
+apt-get install -y -q python-pip python-virtualenv libpython-dev git
+
+# Install postgres
+apt-get install -y -q postgresql postgresql-contrib postgresql-server-dev-9.3
+
+# Run postgres at start
+update-rc.d postgresql enable
+service postgresql start
+
+# Create "vagrant" user and "bc" and "test_bc" databases
+sudo -u postgres psql -c "CREATE USER vagrant WITH PASSWORD 'vagrant';"
+sudo -u postgres psql -c "ALTER USER vagrant CREATEDB;"
+sudo -u postgres psql -c "CREATE DATABASE bc;"
+sudo -u postgres psql -c "GRANT ALL PRIVILEGES ON DATABASE bc TO vagrant;"
+sudo -u postgres psql -c "CREATE DATABASE test_bc;"
+sudo -u postgres psql -c "GRANT ALL PRIVILEGES ON DATABASE test_bc TO vagrant;"
+
+# Install memcached
+apt-get install -y -q memcached libmemcached-dev
+
+# Install gettext (used to compile locales)
+apt-get install -y -q gettext
+
+
+# Build virtual environment
+VENV=/home/vagrant/env/bc
+sudo -H -u vagrant virtualenv $VENV
+
+# Update pip and install BC requirements
+sudo -H -u vagrant -s -- <<EOF
+source $VENV/bin/activate
+pip install --upgrade requests
+pip install --upgrade "pip>=8"
+cd ~/src
+pip install -r requirements/development.txt
+EOF
+
+# Remove unused packages, most of them are related to X server
+# and we don't use X server at all.
+apt-get autoremove -y -q
+
+# Create a .env
+cat > /home/vagrant/src/.env <<EOF
+DJANGO_DEBUG=1
+EMAIL_BACKEND="django.core.mail.backends.console.EmailBackend"
+SECRET_KEY="seatec astronomy"
+DATABASE_URL="postgres://vagrant:vagrant@localhost/bc"
+# MEMCACHE_SERVERS='localhost:11211'
+# MEMCACHE_USERNAME=''
+# MEMCACHE_PASSWORD=''
```

Comment:
I have memcache set up in the vagrant environment, but when I set these, then it kicks up errors when running:

```
[10/Feb/2016 16:24:18] "GET / HTTP/1.1" 200 4081
Traceback (most recent call last):
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 86, in run
    self.finish_response()
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 131, in finish_response
    self.close()
  File "/usr/lib/python2.7/wsgiref/simple_server.py", line 36, in close
    SimpleHandler.close(self)
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 259, in close
    self.result.close()
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django/http/response.py", line 309, in close
    signals.request_finished.send(sender=self._handler_class)
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django/dispatch/dispatcher.py", line 189, in send
    response = receiver(signal=self, sender=sender, **named)
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django/core/cache/__init__.py", line 156, in close_caches
    cache.close()
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django/core/cache/backends/memcached.py", line 111, in close
    self._cache.disconnect_all()
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django_pylibmc/memcached.py", line 91, in _cache
    client = self._lib.Client(self._servers, **client_kwargs)
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/pylibmc/client.py", line 143, in __init__
    behaviors=_behaviors_numeric(behaviors))
TypeError: libmemcached does not support SASL
[10/Feb/2016 16:24:18] "GET / HTTP/1.1" 500 59
----------------------------------------
Exception happened during processing of request from ('10.0.2.2', 36034)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 593, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 102, in __init__
    super(WSGIRequestHandler, self).__init__(*args, **kwargs)
  File "/usr/lib/python2.7/SocketServer.py", line 649, in __init__
    self.handle()
  File "/home/vagrant/env/bc/local/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 182, in handle
    handler.run(self.server.get_app())
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 92, in run
    self.close()
  File "/usr/lib/python2.7/wsgiref/simple_server.py", line 33, in close
    self.status.split(' ',1)[0], self.bytes_sent
AttributeError: 'NoneType' object has no attribute 'split'
```

Thus I commented it out. I'm not sure what to do here. Thoughts?